game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -687,14 +687,13 @@ public void setServerLauncher(final ServerLauncher launcher) {
 
   @Override
   public String toString() {
-    final StringBuilder sb = new StringBuilder();
-    sb.append("ServerModel GameData:").append(data == null ? "null" : data.getGameName()).append("\n");
-    sb.append("Connected:").append(serverMessenger == null ? "null" : serverMessenger.isConnected()).append("\n");
-    sb.append(serverMessenger);
-    sb.append("\n");
-    sb.append(remoteMessenger);
-    sb.append("\n");
-    sb.append(channelMessenger);
-    return sb.toString();
+    String sb = "ServerModel GameData:" + (data == null ? "null" : data.getGameName()) + "\n"
```

Comment:
you can return this directly now, no need for the `String sb`; would also be good to get consistency on the new line append formatting.